app-blank-after-kendo-upgrade.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Upgrading from Kendo UI Core to Professional Breaks App
+description: After upgrading an app through the Package Manager to use Kendo UI Professional, it appears blank on start up. 
+type: troubleshooting
+slug: app-broken-after-kendo-upgrade,
+tags: kendo, package-manager, references
+ticketid: 1017633
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package Manager</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+</table>
+## Description
+When you switch between Kendo UI Core and Kendo UI Professional in your app using the **Package Manager**, you may lose all style and functionality. If you check the Simulator or Device console, you will see multiple error messages about missing files and, usually, an "kendo is not defined" error.
+
+## Error Message
+Uncaught ReferenceError: kendo is not defined.
```

Comment:
As per a mail discussion we had, all error messages should be code-formatted.

Use the code block formatting here <code>```</code>